include/private/rdyconfigimpl.h

```diff
@@ -159,10 +159,14 @@ typedef struct {
 
 // all output parameters
 typedef struct {
-  RDyOutputFormat format;       // file format
-  PetscInt        interval;     // output interval [steps between outputs]
-  PetscInt        batch_size;   // number of timesteps per output file (if available)
-  RDyTimeSeries   time_series;  // time series appended to text (.dat) files
+  PetscBool       enable;            // true if output is requested
+  RDyOutputFormat format;            // file format
+  PetscInt        step_interval;     // output interval [steps between outputs]
+  PetscReal       time_interval;     // temporal interval at which output is written
+  RDyTimeUnit     time_unit;         // unit of time for temporal interval
+  PetscInt        batch_size;        // number of timesteps per output file (if available)
+  RDyTimeSeries   time_series;       // time series appended to text (.dat) files
+  PetscReal       prev_output_time;  // previous time at which output was written
```

Comment:
This isn't a config parameter, right? We might want to put this somewhere in the model instead, since it's state information.